lib/std/fs.zig

```diff
@@ -2238,14 +2238,13 @@ pub const Dir = struct {
         return file.stat();
     }
 
-    pub const StatFileError = File.OpenError || StatError;
-
-    // TODO: improve this to use the fstatat syscall instead of making 2 syscalls here.
-    pub fn statFile(self: Dir, sub_path: []const u8) StatFileError!File.Stat {
-        var file = try self.openFile(sub_path, .{});
-        defer file.close();
+    pub fn statFile(self: Dir, sub_path: []const u8) os.FStatAtError!Stat {
+        return Stat.fromSystemStat(try self.fstatat(sub_path));
+    }
 
-        return file.stat();
+    pub fn fstatat(self: Dir, sub_path: []const u8) os.FStatAtError!os.system.Stat {
+        const flags = 0; // no flags
```

Comment:
`no flags` is not very explanatory.
The flag indicates that symlinks are followed, which must be disabled on Linux with `AT_SYMLINK_NOFOLLOW`.

Reply:
I just had no idea what to do with these flags.
Is there another flag I should add?